src/CDFAttribute.php

```diff
@@ -86,8 +86,13 @@ public function getValue() {
     return $this->value;
   }
 
-  public function setValue($value, $langauge = 'und') {
-    $this->value[$langauge] = $value;
+  public function setValue($value, $language = CDFObject::LANGUAGE_UNDETERMINED, $overwrite_all = FALSE) {
```

Comment:
Please see discussion in https://github.com/acquia/content-hub-d8/pull/306/files. The new overwrite_all might not be needed in this story, and could be moved to the `data` attribute story.